features/partnerhistorie.feature

```diff
@@ -31,8 +31,8 @@ Functionaliteit: Tonen van Partnerhistorie
 
   Als een huwelijk/partnerschap, actueel of ontbonden, in onderzoek is, en dit onderzoek is niet afgerond (Datum einde onderzoek is leeg), wordt inOnderzoek gevuld voor betreffende huwelijk/partnerschap.
 
-  Als wel het begin van de periode (datumvan) wordt opgegeven, maar geen einde van de periode (datumtotenmet), dan worden alle huwelijken/partnerschappen vanaf de datumvan in het antwoord opgenomen.
-  Als wel het einde van de periode (datumtotenmet) wordt opgegeven, maar geen begin van de periode (datumtotenmet), dan worden alle huwelijken/partnerschappen tot en met de datumtotenmet in het antwoord opgenomen.
+  Als wel het begin van de periode (datumVan) wordt opgegeven, maar geen einde van de periode (datumTotEnMet), dan worden alle huwelijken/partnerschappen vanaf de datumVan in het antwoord opgenomen.
```

Comment:
In regel 30 staat

'..., zodat de meest laatst gesloten huwelijk/partnerschap bovenaan staat.'

Waarom niet 

'..., zodat het laatst gesloten huwelijk/partnerschap bovenaan staat.'

Reply:
Aangepast